--- edb-1.0.5.004/configure.in.orig	2005-07-30 08:03:20.000000000 +0200
+++ edb-1.0.5.004/configure.in	2005-08-24 13:54:09.000000000 +0200
@@ -24,55 +24,6 @@
 dnl AC_PROG_INSTALL()
 dnl AC_ARG_PROGRAM()
 
-dnl This is where we handle stuff that autoconf can't handle: compiler,
-dnl preprocessor and load flags, libraries that the standard tests don't
-dnl look for.  The default optimization is -O.
-dnl
-dnl XXX
-dnl Don't override anything if it's already set from the environment.
-AC_SUBST(CFLAGS)
-AC_SUBST(CPPFLAGS)
-dnl AC_SUBST(CXX)
-dnl AC_SUBST(CXXFLAGS)
-AC_SUBST(LDFLAGS)
-case "$host_os" in
-aix4.1*)   CFLAGS=${CFLAGS-"-O3"};;
-bsd4.4)    CFLAGS=${CFLAGS-"-O2"};;
-bsdi4*)	   CFLAGS=${CFLAGS-"-O2"};;
-bsdi*)	   CC=${CC-"shlicc2"}
-	   CFLAGS=${CFLAGS-"-O2"}
-	   LIBS="-lipc $LIBS";;
-irix*)	   CFLAGS=${CFLAGS-"-O2"};;
-nextstep3) CFLAGS=${CFLAGS-"-O2"};;
-linux*)	   CFLAGS=${CFLAGS-"-O2"};;
-mpeix*)	   CPPFLAGS="-D_POSIX_SOURCE -D_SOCKET_SOURCE $CPPFLAGS"
-	   LIBS="-lsocket -lsvipc $LIBS";;
-sco*)	   CFLAGS="$CFLAGS -b elf"
-	   LIBS="-lsocket -lnsl $LIBS";;
-*)	   CFLAGS=${CFLAGS-"-O"};;
-esac
-CXXFLAGS=${CXXFLAGS-"$CFLAGS"}
-
-dnl Optional C++ API..
-dnl
-dnl XXX
-dnl Versions of GCC up to 2.8.0 required -fhandle-exceptions, but it is
-dnl the default in versions 2.8.0 and after.
-dnl AC_MSG_CHECKING(if --enable-cxx option specified)
-dnl AC_ARG_ENABLE(cxx,
-dnl	[  --enable-cxx            Provide C++ interfaces.],
-dnl	[db_cv_cxx="$enable_cxx"], [db_cv_cxx="no"])
-dnl AC_MSG_RESULT($db_cv_cxx)
-dnl if test "$db_cv_cxx" = "yes"; then
-dnl	AC_PROG_CXX
-
-dnl	if test "$GXX" = "yes"; then
-dnl		CXXFLAGS="-fhandle-exceptions $CXXFLAGS"
-dnl	fi
-
-dnl	AC_LIBOBJ="$AC_LIBOBJ \$(COBJS)"
-dnl fi
-
 dnl There are additional libraries we need for some compiler/architecture
 dnl combinations.
 AC_SUBST(LIBS)
@@ -404,7 +355,7 @@
 ncurses_ok=no
 gtk_ok=no
 
-GTK_CONFIG=`which gtk-config`
+GTK_CONFIG=/usr/bin/gtk-config
 if test -x "$GTK_CONFIG"; then 
   AC_CHECK_LIB(gtk, gtk_init, gtk_ok=yes, gtk_ok=no, `$GTK_CONFIG --libs`)
 fi
@@ -456,7 +407,6 @@
 edb-native.oe
 edb.bb
 edbXnative.bb
-debian/changelog
-], [
-chmod +x edb-config
-])
+debian/changelog])
+AC_CONFIG_COMMANDS([default],
+	[chmod +x edb-config])
